public/app/features/alerting/partials/alert_tab.html

```diff
@@ -82,14 +82,23 @@ <h5 class="section-heading">Conditions</h5>
 
 			<div class="gf-form-group">
 				<div class="gf-form">
-          <span class="gf-form-label">If no data points or all values are null</span>
+          <span class="gf-form-label width-18">If no data points or all values are null</span>
           <span class="gf-form-label query-keyword">SET STATE TO</span>
 					<div class="gf-form-select-wrapper">
 						<select class="gf-form-input" ng-model="ctrl.alert.noDataState" ng-options="f.value as f.text for f in ctrl.noDataModes">
 						</select>
 					</div>
 				</div>
 
+				<div class="gf-form">
+          <span class="gf-form-label width-18">If query exeuction throws an error</span>
```

Comment:
Execution is misspelt, and `throws error` seems a bit too programmer speak. Maybe  `On execution error or timeout`, and then the same for `On no data points ..."